src/runtime_src/core/pcie/linux/shim.h

```diff
@@ -83,6 +83,7 @@ class shim
     int xclExportBO(unsigned int boHandle);
     unsigned int xclImportBO(int fd, unsigned flags);
     int xclGetBOProperties(unsigned int boHandle, xclBOProperties *properties);
+    int xclGetMemInfo(char **MemInfo);
```

Comment:
What is this function?

Reply:
Now this function is not required. Getting the group info from the xclbin directly.